### PR TITLE
Fixes and improves #2766

### PR DESCRIPTION
**Pull request by margarett**
Fixes the text about where the membergroup icons should be placed and adds a fallback in Load.php so that, if the icon doesn't exist in the current theme's "membericons" folder, falls back to default

Signed-off-by: Bruno Alves bruno.m.alves@gmail.com

### CODE REVIEW

File: Themes/default/languages/ManageMembers.english.php [hunk 1]
```diff
@@ -79,7 +79,7 @@
 $txt['membergroups_settings'] = 'Membergroup Settings';
 $txt['groups_manage_membergroups'] = 'Groups allowed to change membergroups';
 $txt['membergroups_select_permission_type'] = 'Select permission profile';
-$txt['membergroups_images_url'] = 'Themes/{theme}/images/membericons/';
+$txt['membergroups_images_url'] = 'Themes/default/images/membericons/';
```
Comment (live627) [inline Themes/default/languages/ManageMembers.english.php:82]: Might want to revert this change.

**Comment by margarett** [comment]
@live627 check the discussion here
https://github.com/SimpleMachines/SMF2.1/issues/2766

I don't know if this was the intended behavior or not (although it makes MUCH sense to me as it is) but the suggested text is adequate to how it works ;)
